src/app/features/public-form/public-form-config.ts

```diff
@@ -1,12 +1,13 @@
 import { Entity } from "../../core/entity/model/entity";
 import { DatabaseEntity } from "../../core/entity/database-entity.decorator";
 import { DatabaseField } from "../../core/entity/database-field.decorator";
+import { FieldGroup } from "app/core/entity-details/form/field-group";
 
 @DatabaseEntity("PublicFormConfig")
 export class PublicFormConfig extends Entity {
   @DatabaseField() title: string;
   @DatabaseField() description: string;
   @DatabaseField() entity: string;
-  @DatabaseField() columns: string[][];
+  @DatabaseField() columns: FieldGroup;
   @DatabaseField() prefilled: { [key in string]: any };
```

Comment:
```suggestion
```

This is not supported anymore, is it?

Reply:
Still we are using this on the migration which we have added for backward compatibility in publicformcomponent, by checking if this is present remove this from config.

Reply:
ah, I understand. But if the current/new config format doesn't allow this anymore without migration, let's remove it from this class. (for the migration method, you can still access the old field, e.g. using `formConfig["prefilled"]`)

Reply:
Sorry, turns out we need this afterall - otherwise the EntityMapperService just skips this property when loading old configs from the DB. I added it again as "deprecated"